projects/lib/src/board/twokingseachboard.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+    This file is part of Cute Chess.
+
+    Cute Chess is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Cute Chess is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Cute Chess.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "twokingseachboard.h"
+#include "westernzobrist.h"
+
+namespace Chess {
+
+TwoKingsEachBoard::TwoKingsEachBoard()
+	: WesternBoard(new WesternZobrist()),
+	  m_symmetrical(isSymmetrical()),
```

Comment:
OK, I think just like that. I wondered why the compiler ate this. When I coded this into the constructor I was fully expecting an error message or undefined behaviour at run time - but, no.

Thank you for the feedback. Being uneasy with the code myself, I will gladly change the value to a false in the constructor.